tests/integration/test_kfp_functional_v1.py

```diff
@@ -80,28 +79,37 @@ async def test_build_and_deploy(ops_test: OpsTest, request, lightkube_client):
     bundlefile_path = Path(request.config.getoption("bundle"))
     basedir = Path("./").absolute()
 
-    # Build the charms we need to build only if --no-build is not set
     context = {}
-    if not no_build:
+
+    # Find charms in the expected path if `--charms-path` is passed
+    if charms_path := request.config.getoption("--charms-path"):
+        for charm in KFP_CHARMS:
+            cached_charm = f"{charms_path}/{charm}/{charm}_ubuntu@20.04-amd64.charm"
```

Comment:
Let's add a comment here that the full path is more or less hardcoded and that it depends on the downloaded artefacts.

Reply:
commented